sme.gclass.api.dados/Repositorios/Eol/Usuarios/RepositorioAlunoEol.cs

```diff
@@ -651,5 +651,73 @@ ORDER BY
 					FROM
 						#tempAlunosMatriculasAtivasFinal temp;";
         }
+
+        public async Task<PaginacaoResultadoDto<long>> ObterCodigosAlunosInativosPorAnoLetivo(Paginacao paginacao, int anoLetivo)
+        {
+            try
+            {
+				using var conn = ObterConexao();
+
+				var querySelectDados = @" SELECT DISTINCT a.cd_aluno AS CodigoAluno ";
+
+				var querySelectCount = "SELECT COUNT(DISTINCT a.cd_aluno) ";
+
+				var queryFrom = new StringBuilder(@" FROM
+											v_aluno_cotic aluno (NOLOCK)
+										INNER JOIN
+											aluno a
+											ON aluno.cd_aluno = a.cd_aluno
+										INNER JOIN
+											v_matricula_cotic matr (NOLOCK)
+											ON aluno.cd_aluno = matr.cd_aluno
+										INNER JOIN
+											matricula_turma_escola mte (NOLOCK)
+											ON matr.cd_matricula = mte.cd_matricula
+										INNER JOIN
+											turma_escola te (NOLOCK)
+											ON mte.cd_turma_escola = te.cd_turma_escola
+										INNER JOIN
+											escola esc (NOLOCK)
+											ON te.cd_escola = esc.cd_escola
+										WHERE
+											matr.st_matricula IN (2,3,4,5,7,8,11,12,14,15)
+											AND mte.cd_situacao_aluno IN (2,3,4,5,7,8,11,12,14,15)
```

Comment:
esse cara estava errado na consulta de remoção de alunos do curso
```suggestion
											mte.cd_situacao_aluno IN (2,3,4,5,7,8,11,12,14,15)
```